### ISSUE TITLE
check jwt implementation

### ISSUE THREAD

**Issue by malteish** [dm3-org/dm3#909]
- [x] Would it be a good idea to use asymmetric keys in our jwt? Are they available both in backend and delivery service? Then do it.
- [x] issue jwt with challenge instead of keeping challenge ich db
- [ ] ~clean up db schema: remove session completely~ -> moved to #907 
- [x] do we need to [check what the ens resolves to](https://github.com/dm3-org/dm3/blob/fd39b0b9e32f0aae0994ab8a395b42d1ef0af458/packages/lib/delivery/src/Session.ts#L34) when checking the existing token? That doesn't actually answer any questions imo.
- [x] ~change auth message definition: why `Bearer` -> remove, make key shorter for efficiency.~ Seems to be standard, so keep it.
- [x] add `nbf` (not before) to token instead of manually verifying `iat`
- [x] check schema of auth token with schema tool instead of manually
- [x] decide for `user` or `account` in both tokens

**Comment by malteish** [comment]
using symmetric encryption is standard and we do it, too

**Comment by malteish** [comment]
concerning the resolving of ens to address: [discussion](https://corpusventures.slack.com/archives/C047JB95ZDZ/p1718264341733259)

tldr: not necessary when only checking an auth token. Removed in 08993c6
